grants/milestone-deliverables-guidelines.md

```diff
@@ -1,28 +1,82 @@
 # Milestone Deliverables Guidelines
 
-We kindly ask you to consider the following guidelines when you submit a milestone for evaluation. Please submit your milestones by email to grants@web3.foundation.
+We kindly ask you to consider the following guidelines when you submit a milestone for evaluation. 
+Please submit your milestones by email to grants@web3.foundation.
 
 The submission should contain the following information: 
  * **Name of the grant project**
  * **Link to the open-source code/delivery** 
- * **Basic [testing guide](#testing-guide)** 
+ * **[License](#license)**
+ * **[Documentation](#documentation)**
+ * **[Testing Guide](#testing-guide)**
+ * **[End-user Tutorial](#end-user-tutorial)**
```

Comment:
I would not make the end-user tutorial mandatory. A testing guide should cover everything